SingularityService/src/main/java/com/hubspot/singularity/mesos/SingularityMesosScheduler.java

```diff
@@ -135,7 +138,7 @@ public void resourceOffers(SchedulerDriver driver, List<Protos.Offer> offers) {
     final Set<Protos.OfferID> acceptedOffers = Sets.newHashSetWithExpectedSize(offers.size());
 
     for (Protos.Offer offer : offers) {
-      slaveAndRackManager.checkOffer(offer);
+      slaveAndRackManager.checkOffer(offer, inactiveSlaveManager.getInactiveSlaves());
```

Comment:
For efficiency we should probably grab this before looping through offers so we don't grab it 50+ times